docs/cip/README.md

```diff
@@ -548,160 +1192,555 @@ A detailed cost analysis of Leios deployment is documented in [Leios node operat
-- [ ]  Audit.
-- [ ]  Successful operation in testnet environments.
-- [ ]  Community agreement on the settings for the Leios protocol parameters.
+>
+> Describes what are the acceptance criteria whereby a proposal becomes
+> _'Active'_.
+>
+> This sub-section must define a list of criteria by which the proposal can
+> become active. Criteria must relate to observable metrics or deliverables and
+> be reviewed by editors and project maintainers when applicable. For example:
+> "The changes to the ledger rules are implemented and deployed on Cardano
+> mainnet by a majority of the network", or "The following key projects have
+> implemented support for this standard".
+
+- [ ] The revised `cardano-node` implementations pass the node-level conformance
+      test suites.
+- [ ] Audit.
+- [ ] Successful operation in testnet environments.
+- [ ] Community agreement on the settings for the Leios protocol parameters.
 
 ### Implementation plan
 
-> [!NOTE]
-> Either a plan to meet those criteria or `N/A` if not applicable.
-> 
-> This sub-section should define the plan by which a proposal will meet its acceptance criteria, when applicable. More, proposals that require implementation work in a specific project may indicate one or more implementors. Implementors must sign off on the plan and be referenced in the document's preamble.
-> 
-> In particular, an implementation that requires a hard-fork should explicitly mention it in its _'Implementation Plan'_.
+> [!NOTE] Either a plan to meet those criteria or `N/A` if not applicable.
+>
+> This sub-section should define the plan by which a proposal will meet its
+> acceptance criteria, when applicable. More, proposals that require
+> implementation work in a specific project may indicate one or more
+> implementors. Implementors must sign off on the plan and be referenced in the
+> document's preamble.
+>
+> In particular, an implementation that requires a hard-fork should explicitly
+> mention it in its _'Implementation Plan'_.
 
-- [ ]  Detailed node-level (as opposed to this protocol-level) specification.
-- [ ]  Develop node-level conformance test suite.
-- Consider developing a "quick and dirty" implementation for large scale experiments.
+- [ ] Detailed node-level (as opposed to this protocol-level) specification.
+- [ ] Develop node-level conformance test suite.
+- Consider developing a "quick and dirty" implementation for large scale
+  experiments.
 - Coordinate with related activities on other protocol enhancements.
-    - Compatibility between Peras, Leios, and Genesis.
-    - Common design and implementation for certificates, voting, and related key registration: Mithril, Peras, Leios, and partner chains.
-- Triage by intersect Core Infrastructure, Consensus, Ledger, and Network functions.
+  - Compatibility between Peras, Leios, and Genesis.
+  - Common design and implementation for certificates, voting, and related key
+    registration: Mithril, Peras, Leios, and partner chains.
```

Comment:
https://github.com/berewt/CIPs/blob/master/CIP-xxxx/CIP-xxxx.md is relevant for key registration.